.github/workflows/build-push-default-reports-build.yml

```diff
@@ -0,0 +1,60 @@
+name: Report Builds
+on:
+  push:
+    branches: 
+    - main
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  REPO_NAME: open-metrics-builds
+  REGISTRY_NAME: public.ecr.aws/resim
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.ECR_RW_ROLE }}
+          aws-region: us-east-1
+
+      - name: Log in to ECR Public
+        uses: docker/login-action@v2
+        with:
+          registry: public.ecr.aws
+      
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Docker metadata
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ env.REGISTRY_NAME }}/${{ env.REPO_NAME }}
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=sha
+            
+      - name: Build
+        env:
+          REPO_NAME: ${{ env.REGISTRY_NAME }}/${{ env.REPO_NAME }}
+          IMAGE_VERSION: ${{ steps.meta.outputs.version }}
+          SHA: ${{ github.sha }}
+        run: bazel build //resim/metrics/default-report-metrics:default_report_metrics_build
+
+      - name: Push
+        if: ${{ github.ref == 'refs/heads/main' }}
+        env:
+          REPO_NAME: ${{ env.REGISTRY_NAME }}/${{ env.REPO_NAME }}
+          IMAGE_VERSION: ${{ steps.meta.outputs.version }}
+          SHA: ${{ github.sha }}
+        run: bazel run //:default_report_metrics_build_push -- --repository "$REPO_NAME" --tag "$IMAGE_VERSION" --tag "sha-${SHA}"
```

Comment:
I think we need to be careful about how we do this. As it currently stands, it seems like there's nothing in the full URI of the image being pushed to identify it as a report metrics build and if we push other images their tags will collide unless we differentiate. Maybe `--tag "report_metrics_${IMAGE_VERSION}"` or something like that?

Reply:
good point. moved to using namespacing to keep the tag clean